doc/manuals/admin/perf_tuning.md

```diff
@@ -40,19 +40,28 @@ speeds up reads) and write efficiency (the usage of indexes slows down writes) a
 consume space in database and mapped RAM memory) and it is the administrator (not Orion) who has
 to decide what to prioritize.
 
-However, in order to help administrators in this task, the following indexes are recommended:
+However, in order to help administrators in this task, the following *general* indexes are recommended:
 
 * Collection [entities](database_model.md#entities-collection)
-    * `_id.id`
-    * `_id.type`
-    * `_id.servicePath`
-    * `attrNames`
+    * `{_id.servicePath: 1, _id.id: 1, _id.type: 1}` (note this is a compound index and key order matters in this case)
```

Comment:
note:
      OR
note that ...

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3495/commits/693187af336cc4a95a919d0b5e8eba2807ae05fd

Reply:
Sorry, I wasn't clear in my comment ...
note that this is a ...

Reply:
Fixed in 6efa4d8e596c4271e7fc7ddf7fbac7aea3b92679